eng/Packages.Data.props

```diff
@@ -57,6 +57,15 @@
   <!--
     Dependency versions for Track 2, Azure.* libraries.
     Only packages that are approved dependencies should go here.
+
+    Prior to making any updates to this section, potential version
+    conflicts with PowerShell or Functions need to be investigated.
+    
+    For any common dependencies with PowerShell, the most straightforward
```

Comment:
For Functions, this file can be referenced - https://github.com/Azure/azure-functions-host/blob/dev/test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json

Versions cannot be higher than those listed in the file.

Reply:
Added! Let me know if it looks good.

Reply:
Looks good!